dev/gendoc.py

```diff
@@ -19,21 +19,20 @@
 Contains useful utils to generate reStructuredText files for Sphinx.
 """
 
+import json
+import os
+import platform
+import re
 import shutil
 import sys
 from distutils.version import LooseVersion
 from urllib.error import HTTPError
 from urllib.request import urlopen, Request
-import json
-import os
-import re
-import platform
 
+import databricks.koalas as ks
 import pypandoc
 from pypandoc import pandoc_download
 
-import databricks.koalas as ks
```

Comment:
and PEP8 specifies the import order. I believe the previous import order was correct. Let's revert this change back.